Formula/freeciv.rb

```diff
@@ -79,7 +79,7 @@ def install
 
   test do
     system bin/"freeciv-manual"
-    assert_predicate testpath/"classic6.mediawiki", :exist?
+    assert_predicate testpath/"civ2civ36.mediawiki", :exist?
```

Comment:
https://github.com/freeciv/freeciv/blob/R3_0_1/data/classic/README.classic
> Classic ruleset was freeciv's default ruleset until freeciv version 3.0,
where it got replaced by civ2civ3 ruleset.